conbench/entities/history.py

```diff
@@ -468,6 +472,8 @@ def _add_rolling_stats_columns_to_df(
     exclusive on the right side. This is useful if you want to compare each commit to
     the previous commit's rolling stats.
     """
+    df = zscore_automated_detect(df=df)
```

Comment:
Going from top to bottom (I have not looked at the implementation of the function below), I think we can find a more expressive name than `zscore_automated_detect()`. If this is about outlier detection, then maybe put that in the name?

Reply:
Yeah, didn't love this. Trying `_detect_shifts_with_trimmed_estimators()`—this method really turns more on the trimming than the z-scores